albumentations/core/composition.py

```diff
@@ -167,6 +167,9 @@ def __call__(self, force_apply=False, **data):
             p.ensure_data_valid(data)
         transforms = self.transforms if need_to_run else self.transforms.get_always_apply(self.transforms)
         dual_start_end = transforms.start_end if self.processors else None
+        check_each_transform = any(
+            getattr(item.params, "check_each_transform", False) for item in self.processors.values()
+        )
```

Comment:
It looks like that if `check_each_transform` is set to `True` in any processor, then it will affect the behavior of all processors, and they all start to filter values at each step. If this a global setting that affects all processors, maybe we should set it in a separate place and not in a processor?

Reply:
Now this parameter will be checked only for selected processor.